hazelcast/src/main/java/com/hazelcast/wan/impl/WanEventContainerReplicationOperation.java

```diff
@@ -69,7 +69,7 @@ public void run() throws Exception {
         int partitionId = getPartitionId();
 
         for (WanReplicationConfig wanReplicationConfig : wanReplicationConfigs) {
-            service.appendWanReplicationConfig(wanReplicationConfig);
+            service.addWanReplicationConfigLocally(wanReplicationConfig);
```

Comment:
This is the only part which needs to be forward-ported to 4.1, with an adapted test in EE.